azkaban-db/src/main/java/azkaban/db/DatabaseSetup.java

```diff
@@ -190,11 +120,8 @@ public boolean accept(final File pathname) {
 
       final String name = pathname.getName();
       final int length = name.length();
-      if (this.suffix.length() > length || this.prefix.length() > length) {
-        return false;
-      }
-
-      return name.startsWith(this.prefix) && name.endsWith(this.suffix);
+      return !(this.suffix.length() > length || this.prefix.length() > length) && name
```

Comment:
why not this.suffix.length() <= length && this.prefix.length() <= length

Reply:
Good find. I was adopting Intellij's suggestion from previous code, and not carefully checking that.

Reply:
Modified.